libs/akita/src/__tests__/setLoading.spec.ts

```diff
@@ -28,13 +28,13 @@ describe('setLoading', () => {
     expect(query.getValue().loading).toBe(false);
     cacheable(store, request).subscribe(spy);
     // from inside withLoading
-    expect(store.setLoading).toBeCalledTimes(2);
```

Comment:
Why the change?

Reply:
The current behaviour is, that `EntityStore.set()` set `hasCache` of the store to true. I remove it, because entities are treated separately from the store: https://github.com/datorama/akita/pull/471/files#diff-be97ad95765300e4c32ef78919456d39R141

The `setLoading` call count changed as a result of `cacheable` is triggered differently because `hasCache` isn't set any more.